api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -389,19 +390,32 @@ private User initializeUserIfNeeded() {
   }
 
   @Override
-  public ResponseEntity<Profile> submitIdVerification(IdVerificationRequest request) {
-    // TODO(dmohs): Prevent this if the user has already attempted verification?
-    Person person = blockscoreService.createPerson(
-      request.getFirstName(), request.getLastName(),
-      new Address()
-        .setStreet1(request.getStreetLine1()).setStreet2(request.getStreetLine2())
-        .setCity(request.getCity()).setSubdivision(request.getState())
-        .setPostalCode(request.getZip()).setCountryCode("US"),
-      request.getDob(),
-      request.getDocumentType(), request.getDocumentNumber()
-    );
+  public ResponseEntity<Profile> submitIdVerification() {
```

Comment:
Are you separately planning on adding this field to Profile / API so the UI can show when email verification request has been sent?